cyberark-secrets-provider-for-kubernetes/values.yaml

```diff
@@ -0,0 +1,36 @@
+# K8s permission defaults
```

Comment:
@InbalZilberman see below comments. I would like Shuli to take a look at the comments here as well. I will open a card

Reply:
Thanks